docs/ssl.md

```diff
@@ -0,0 +1,19 @@
+# Cryptography Libraries
+
+The underlying cryptography libraries describe [here](../README.md#ssl-compatibility) can be selected when configuring CMake by
+explicity setting `JWT_SSL_LIBRARY` to one of three values
+
+- OpenSSL
+- LibreSSL
+- wolfSSL
+
+Here's an example:
+
+```sh
+cmake . -DJWT_SSL_LIBRARY:STRING=wolfSSL 
+```
+
+## Notes
+
+JWT-CPP relies on the OpenSSL API, as a result both LibreSSL and wolfSSL need to include their respective compatability layers.
+Most system already have OpenSSL so it's important to make sure when compiling you application only the correct one is used.
```

Comment:
```suggestion
Most system already have OpenSSL so it's important to make sure when compiling your application it only includes one. Otherwise you may have missing symbols when linking.
```